docs/UserGuide.adoc

```diff
@@ -93,63 +93,27 @@ The name you add will be automatically formatted. (E.g. "jane doe" will become "
 ====
 
 Example: +
+Suppose you want to add your classmate named Jane Doe, with phone number 12345678 and email address janedoe@gmail.com
+into NOVA,
+
 `add n\Jane Doe p\12345678 e\\janedoe@gmail.com c\classmate`
 
-Adds a new contact with name Jane Doe, phone number 12345678 and email address janedoe@gmail.com into the
+NOVA will add a new contact named Jane Doe, phone number 12345678 and email address janedoe@gmail.com into the
 classmate category.
 
-==== Edit Contact: `edit`
-You can edit the contacts that you have added. If the contact you want to edit does not exist, NOVA will let you know.
-
-At least one of the optional fields must be provided. Optional fields are `n\[name]`, `p\[phone number]`,
-`e\[email address]` or `c\[classmate/teammate]`.
-
-Format: `edit i\[index] n\[name] p\[phone number] e\[email address] c\[classmate/teammate]`
-
-[NOTE]
-====
-* Edits the person at the specified `[index]` in the displayed person list
-* The `[index]` must be a positive integer. (E.g. 1, 2, 3, ...)
-* `[name]` and `[classmate/teammate]` are not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe") +
-The name you add will be automatically formatted. (E.g. "jane doe" will become "Jane Doe")
-* You can only edit either `classmate` or `teammate` as category
-====
-
-Example:
-
-* `edit i\1 p\88888888` +
-Edits the phone number of the first person in your address book to 88888888.
-
-* `edit i\3 e\\kly838@hotmail.com` +
-Edits the email address of the third person in your address book to kly838@hotmail.com.
-
-==== Delete Contact : `delete`
-You can delete a contact that you have added. If the contact you try to delete does not exist, NOVA will let you know.
-
-Format: `delete i\[index]`
+==== List all Contacts: `list`
+You can list the contact's name, phone number and category of all contacts.
 
-[NOTE]
-====
-* Deletes the person at the specified `[index]` in the displayed person list
-* The `[index]` must be a positive integer. (E.g. 1, 2, 3, ...)
-====
+Format: `list`
 
 Example: +
-`delete i\1`
-
-Deletes the first person in your address book.
-
-==== Clear all Contacts: `clear`
-You can clear the contacts that you have added in your address book. If there is no contact, NOVA will let you know.
+Suppose you want to view all your contacts that you have added into Nova,
 
-Format: `clear`
+`list`
 
-==== List all Contacts: `list`
-You can list the contact's name, phone number and category of all contacts.
-
-Format: `list`
+Nova will show all your contacts.
```

Comment:
If there are no parameters, I think there's no need for example since it will just be the same as format.